app/assets/stylesheets/components/_buttons.scss

```diff
@@ -0,0 +1,77 @@
+.primary-button {
+  border: solid 2px black;
+  // background-color: #FFBE09;
+  border-radius: 50px;
+  padding: 8px;
+  width: 200px;
+  justify-content: center;
+  font-weight: bold;
+  display: inline-block;
+  text-align: center;
+}
+
+.primary-button:hover {
+  background-color: #ededed;
+}
+
+
+.secondary-button {
+  border: solid 1px grey;
+  background-color: #ededed;
```

Comment:
You can decide to extract the colours in into settings file so you can easily change colours later. not a dealbreaker but I do it myself a lot